src/Modal.tsx

```diff
@@ -196,8 +170,13 @@ export class ModalManager extends TypedEventEmitter<ModalManagerEvent, HandlerMa
         this.reRender();
     }
 
+    /**
+     * @typeParam A - the arguments to the onFinished callback
+     * @typeParam P - the props of the component
```

Comment:
are A and P actually used here?

Reply:
Whoops this comment is stale from when I tried to make the types even more rigid and failed

```suggestion
```